### PR TITLE
small refactor

### PR DESCRIPTION
**Pull request by datawolf**
Signed-off-by: Wang Long <long.wanglong@huawei.com>

### CODE REVIEW

File: libcontainer/cgroups/utils.go [hunk 1]
```diff
@@ -23,33 +23,8 @@ const (
 
 // https://www.kernel.org/doc/Documentation/cgroup-v1/cgroups.txt
 func FindCgroupMountpoint(subsystem string) (string, error) {
-	// We are not using mount.GetMounts() because it's super-inefficient,
-	// parsing it directly sped up x10 times because of not using Sscanf.
-	// It was one of two major performance drawbacks in container start.
```
Comment (hqhq) [inline libcontainer/cgroups/utils.go]: Keep these comments please.

Comment (datawolf) [inline libcontainer/cgroups/utils.go]: OK, I move it in func `FindCgroupMountpointAndRoot`

**Comment by datawolf** [comment]
ping @hqhq

**Comment by dqminh** [comment]
LGTM

**Comment by hqhq** [comment]
LGTM

**Pull request merged**